src/data/adventures.txt

```diff
@@ -414,6 +414,7 @@ Crimbo22	adventure=559	DiffLevel: mid Env: indoor Stat: 0 nowander	Crimbo Train
 Crimbo22	adventure=560	DiffLevel: mid Env: indoor Stat: 0 nowander	Crimbo Train (Passenger Car)
 Crimbo22	adventure=561	DiffLevel: mid Env: indoor Stat: 0 nowander	Crimbo Train (Dining Car)
 Crimbo22	adventure=562	DiffLevel: mid Env: indoor Stat: 0 nowander	Crimbo Train (Coal Car)
+Crimbo22	place=crimbo22_engine	DiffLevel: mid Env: indoor Stat: 0 nowander	Crimbo Train (Locomotive)
```

Comment:
as a "place", I'm not sure if it has an environment. and it probably doesn't appear for autumnaton either, so we can't know the difflevel. Still, this is probably fine.

Reply:
Ah. Yeah, It was copy/paste/modify and I didn't even think about it.

If there is another comment which requires an edit, I will fix it.